### ISSUE TITLE
works with electron 3.0.9 but not with 5.0.0

### ISSUE THREAD

**Issue by ddelapasse2** [LukasMarx/angular-electron#2]
The repo download worked great, but when I tried to copy this process into my electron 5.0.0 project this line fails:

`this.ipc = (<any>window).require("electron").ipcRenderer;`

Is there some updated way to get a reference to the ipcRenderer from inside Angular?

**Comment by LukasMarx** [comment]
Thanks for your issue!

I was able to reproduce the problem. Seems like the additional node features (like require) need to be enabled before they can be used using the `nodeIntegration` flag.

```ts
  win = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: { nodeIntegration: true }
  });
```

The flag was always there but its default value was changed to false in a recent release.

Fixed in 47d6002f8f13935ff62d71adc21828f371b1eb67.

**Comment by ddelapasse2** [comment]
Confirmed fixed!  Thanks for the excellent tutorial and the super fast
reply.

If you have any insight into making this combo of Electron/Ang7/TypeScript
run in the VSCode debugger that's my only remaining struggle.  The debugger
isn't able to find the corresponding Javascript files even when I set the
OutFiles param to the absolute location under electron/dist/*.js.

thanks again,
Deanna
